lib/src/widgets/chat_bubble_widget.dart

```diff
@@ -218,6 +221,7 @@ class _ChatBubbleWidgetState extends State<ChatBubbleWidget> {
                     child: _messagesWidgetColumn(messagedUser),
                   ),
           ),
+          if (isMessageBySender) ...[SizedBox(child: getReciept())],
```

Comment:
We have provided one feasibility called `FeatureActiveConfig`, which is used for enabling and disabling specific features. So, could you please implement the same for these two features?

Reply:
Okay